pkg/virt-operator/install-strategy/create.go

```diff
@@ -3358,3 +3264,56 @@ func createOrUpdateKubeVirtCAConfigMap(
 	}
 	return nil, nil
 }
+
+func removeKvServiceAccountsFromDefaultSCC(targetNamespace string,
+	clientset kubecli.KubevirtClient,
+	stores util.Stores) error {
+
+	modified := false
+
+	SCCObj, exists, err := stores.SCCCache.GetByKey("privileged")
+	if !exists {
+		return nil
+	} else if err != nil {
+		return err
+	}
+
+	SCC, ok := SCCObj.(*secv1.SecurityContextConstraints)
+	if !ok {
+		return fmt.Errorf("couldn't cast object to SecurityContextConstraints: %+v", SCCObj)
+	}
+
+	oldUsers := SCC.Users
+	SCCCopy := SCC.DeepCopy()
```

Comment:
This copy is redundant IMO. You can just initialize an empty `users` list and just modify it whenver you remove a user.

Reply:
@danielBelenky Thank you. Done.